tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -317,6 +319,50 @@ public async Task WhileAwaitingJobTriggeringInstantJobShouldAnywayTriggerCronJob
         Assert.Equal(ExecutionState.OrchestrationCompleted, instantOrchestrationEvents[6].State);
         Assert.Equal(7, instantOrchestrationEvents.Count);
 
+        /*
```

Comment:
@linkdotnet Here are the captured timestamps.

I slightly puzzled by the 24.04 arm result (first below). And the tests don't fail consistently.

Reply:
Compared to the failing one

```
24.04 arm
s 0 2000-01-01T00:00:00.0000000+00:00
i 0 2000-01-01T00:00:01.0000000+00:00
s 3 2000-01-01T00:00:01.0000000+00:00
i 2 2000-01-01T00:00:01.0000000+00:00   <-----
s 4 2000-01-01T00:00:01.0000000+00:00
i 6 2000-01-01T00:00:01.1924400+00:00
```

Given **i 6** entries values aren't that far one from from another, I can't explain why **i 2** hasn't moved forward even a little bit in that case.

Reply:
I've thought about potential concurrency related issues with the FakeTimeProvider, but that seems out of context now that we're running xUnit v3 (https://github.com/dotnet/extensions/blob/main/src/Libraries/Microsoft.Extensions.TimeProvider.Testing/README.md#xunit-v3)

Reply:
Maybe we can give NCronJob some time here?
```csharp
 FakeTimer.Advance(TimeSpan.FromSeconds(1));

// Added here not in the source code:
// To enable processing time
await Task.Delay(100);

 Guid instantOrchestrationId = provider.GetRequiredService<IInstantJobRegistry>().RunInstantJob<SimpleJob>(token: CancellationToken);
```

Reply:
@linkdotnet Ok. I've found a workaround.

Below the latest timings from 24.04 arm
```
s 0 2000-01-01T00:00:00.0020000+00:00
i 0 2000-01-01T00:00:00.0080000+00:00
s 3 2000-01-01T00:00:00.0100000+00:00
i 2 2000-01-01T00:00:00.0603800+00:00
s 4 2000-01-01T00:00:00.0100000+00:00
i 6 2000-01-01T00:00:00.2286500+00:00
```

I've tweaked the setup of  `FakeTimer` to benefit from [AutoAdvanceAmount](https://github.com/dotnet/extensions/blob/main/src/Libraries/Microsoft.Extensions.TimeProvider.Testing/FakeTimeProvider.cs#L59). This more or less simulates time passing between clock access.